charms/kserve-web-app/metadata.yaml

```diff
@@ -9,6 +9,12 @@ description: |
 summary: |
   KServe Web UI.
 
+requires:
+  ingress:
+    interface: ingress
+    schema: https://raw.githubusercontent.com/canonical/operator-schemas/master/ingress.yaml
+    versions: [v1]
+
 containers:
   kserve-web-app:
```

Comment:
Sorry didn't have a better place to suggest this...

Is there a reason we use the name `kserve-web-app` for this charm instead of `kserve-models-web-app`?  If there's a reason then lgtm, but if not it would be good to keep consistent with upstream.  For example, in debugging this PR today I spent some time looking through upstream manifests for `kserve-web-app` before noticing the difference.  I expect anyone new to this charm, maybe updating it for future version bumps, will do the same.

Reply:
It was mainly to make the name shorter, but I think you have good point we should probably just change it. Please feel free to push to this PR and change it.